beziers.scad

```diff
@@ -437,10 +437,8 @@ function bezier_curvature(curve, u) =
 //   move_copies(bezier_curve(bez, 8)) sphere(r=1.5, $fn=12);
 //   trace_bezier(bez, N=len(bez)-1);
 function bezier_curve(curve,n,endpoint=true) =
-      [
-         each bezier_points(curve, rangex(endpoint?n-1:n,0,1)),
-         if (endpoint) last(curve)
-      ];
+    bezier_points(curve, lerpn(0,1,n,endpoint));
```

Comment:
I believe this change makes the final point no longer exact.

Reply:
`lerpn()` is guaranteed exact at the end if `endpoint=true`.

Reply:
lerpn is exact but bezier_points is not.

Reply:
I believe `bezier_points()` is also exact at `u=1`.

Reply:
I thought I had tested this and found otherwise...but now I'm not finding a failure to be exact.